test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -24,6 +24,42 @@
 #include <nntrainer_internal.h>
 #include <nntrainer_test_util.h>
 
+/**
+ * @brief Compare the training statistics
+ */
+static void nntrainer_capi_model_comp_metrics(ml_train_model_h model,
+                                              float train_loss,
+                                              float valid_loss,
+                                              float valid_accuracy) {
+  int status = ML_ERROR_NONE;
+  char *summary = nullptr;
+
+  /** Compare training statistics */
+  status = ml_train_model_get_summary(
+    model, (ml_train_summary_type_e)ML_TRAIN_SUMMARY_MODEL_TRAIN_LOSS,
+    &summary);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  EXPECT_FLOAT_EQ(std::atof(summary), train_loss);
+  free(summary);
+
+  status = ml_train_model_get_summary(
+    model, (ml_train_summary_type_e)ML_TRAIN_SUMMARY_MODEL_VALID_LOSS,
+    &summary);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  EXPECT_FLOAT_EQ(std::atof(summary), valid_loss);
```

Comment:
if given argument is invalid, `std::atof` has undefined behavior. 
how about using `std::stod`?

Reply:
done.